### ISSUE TITLE
Date validator cannot handle valid timestamps unless they are cast to string

### ISSUE THREAD

**Issue by chetzof** [Respect/Validation#683]
Does not pass validation
```
Validator::date('U')->validate(time());
```

Passes validation
```
Validator::date('U')->validate((string)time());
```

**Comment by batusa** [comment]
thanks for reporting that @chetzof ! I'll have a look at it soon.

**Comment by henriquemoody** [comment]
Fixed on versions 1.0.13 and 1.1.8.

Thanks for reporting! 🐼
